ext/page_log/palm/palm.c

```diff
@@ -467,7 +467,7 @@ palm_complete_checkpoint_ext(WT_PAGE_LOG *page_log, WT_SESSION *session, uint64_
     }
     PALM_KV_ERR(palm, session, ret);
 
-    if (completed_checkpoint >= started_checkpoint) {
+    if (completed_checkpoint > started_checkpoint) {
```

Comment:
I don't think this is correct.

We're in the `complete_checkpoint` function and both `completed_checkpoint` and `started_checkpoint` come from LMDB. So if those values are the same, that implies we are attempting to complete a checkpoint without starting it.